xonsh/wizard.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, prompt='>>> ', converter=None, show_conversion=False,
 class While(Node):
     """Computes a body while a condition function evaluates to true.
     The condition function has the form cond(visitor=None, node=None) and
-    should return an object that is convertable to a bool. The beg attribute
+    should return an object that is convertible to a bool. The beg attribute
```

Comment:
This phrasing feels awkward to me.  Maybe "...should return an object that can be converted to a bool."?